website/docs/reference/commands/retry.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "About dbt retry command"
+sidebar_label: "retry"
+id: "retry"
+---
+
+`dbt retry` re-executes the last `dbt` command from the node point of failure. If the previously executed `dbt` command was successful, `retry` will finish as a no op. 
+
+Retry works with the following commands:
+
+- [`build`](/reference/commands/build)
+- [`compile`](/reference/commands/compile)
+- [`seed`](/reference/commands/seed)
+- [`snapshot`](/reference/commands/build)
+- [`test`](/reference/commands/test)
+- [`run`](/reference/commands/run)
+- [`run-operation`](/reference/commands/run-operation)
+
+`dbt retry` will execute from the node failures recorded in the run_results.json. Executing `dbt retry` without correcting the previous failures will garner idempotent results.
```

Comment:
Might be a good place to add a glossary entry for idempotent. https://docs.getdbt.com/terms/idempotent which I think would be:
`<Term id="idempotent" />`

Reply:
> `dbt retry` will execute from the node failures recorded in the run_results.json

What is the advice here? Are we asking people to make sure they check run_resuts.json and fix any failures?

Reply:
They're not expected to do anything with the file itself, we just wanted to give a behind the curtains look at how the feature works. It references that json to determine where to start the retry. Does adding the parenthesis help?